### ISSUE TITLE
data exchangeably

### ISSUE THREAD

**Issue by severous** [yromano/cqr#6]
@yromano  In conformal prediction, the regression algorithm must treat the data exchangeably. I want to know whether CQR must treat the data exchangeably.

**Comment by yromano** [comment]
Yes, it treats the points exchangeably if you follow the notebooks.

**Comment by severous** [comment]
Does CQR based on  split conformal predictin not apply to non-exchangeable data?
